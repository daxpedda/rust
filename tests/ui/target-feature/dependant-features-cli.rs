//@ revisions: one two three four
//@ only-wasm32-wasip1
//@ compile-flags: --crate-type=lib
//
//
//@ [one] check-fail
//@ [one] compile-flags: -C target-feature=+relaxed-simd
//@ [two] check-fail
//@ [two] compile-flags: -C target-feature=-relaxed-simd,+relaxed-simd
//@ [three] check-fail
//@ [three] compile-flags: -C target-feature=+simd128,+relaxed-simd,-simd128
//@ [four] build-pass
//@ [four] compile-flags: -C target-feature=-simd128,+relaxed-simd -C target-feature=+simd128
