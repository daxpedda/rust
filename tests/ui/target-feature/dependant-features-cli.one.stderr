error: the target feature relaxed-simd requires simd128 to be enabled

error: aborting due to 1 previous error

